3D_print_canairi_code.ino

```diff
@@ -57,18 +55,16 @@ void setup() {
   // If you have a baseline measurement from before you can assign it to start, to 'self-calibrate'
   //sgp.setIAQBaseline(0x8E68, 0x8F41);  // Will vary for each sensor!
 
-   //--------------stepper setup
-   myStepper.setSpeed(60);
+  //--------------stepper setup
+  myStepper.setSpeed(60);
 }
 
-//generic counters and booleans
+// global variables
 int counter = 0;
 int baseLineCounter = 0;
 int calAVG = 0;
 bool birdUP = true;
-int tenSec = 10000;
-int oneMinute = 60000;
-
+const int TEN_SEC = 10000;
```

Comment:
This is a constant, so we might as well declare it as such and let the variable name reflect that :)